velox/functions/sparksql/String.h

```diff
@@ -962,15 +962,8 @@ struct ConvFunction {
   // For signed value, toBase is negative.
   static void
   toChars(out_type<Varchar>& result, int64_t signedValue, int32_t toBase) {
-    int32_t resultSize =
-        (int32_t)std::floor(
-            std::log(std::abs(signedValue)) / std::log(-toBase)) +
-        1;
-    // Negative symbol is considered.
-    if (signedValue < 0) {
-      ++resultSize;
-    }
-    result.resize(resultSize);
+    // 64 + 1, 64 is the max length of int64_t, 1 is for negative symbol.
+    result.resize(65);
     auto toStatus = std::to_chars(
         result.data(), result.data() + result.size(), signedValue, -toBase);
```

Comment:
> base - integer base to use: a value between 2 and 36 (inclusive).

Do we need to ensure `toBase` fits in this range?

Reply:
yeah, it has already ensured at https://github.com/facebookincubator/velox/blob/1b98f0aa277d646632598850d130c48dc6bd5bec/velox/functions/sparksql/String.h#L915-L926